extensions/binding/org.eclipse.smarthome.binding.ntp.test/src/test/java/org/eclipse/smarthome/binding/ntp/test/NtpOSGiTest.java

```diff
@@ -220,7 +219,9 @@ public void testDateTimeChannelTimeZoneUpdate() {
         ZonedDateTime timeZoneFromItemRegistry = ((DateTimeType) getItemState(ACCEPTED_ITEM_TYPE_DATE_TIME))
                 .getZonedDateTime();
         ZoneOffset expectedOffset;
-        if (timeZoneFromItemRegistry.getZone().getRules().isDaylightSavings(timeZoneFromItemRegistry.toInstant())) {
+        String timeZoneInConfiguration = (String) configuration.getProperties().get(NtpBindingConstants.PROPERTY_TIMEZONE);
```

Comment:
I second @sjka's explanation in https://github.com/eclipse/smarthome/pull/5226#issuecomment-376446165 and would vote for not reading out any configuration values but instead use fixed dates. Our string representation in https://github.com/eclipse/smarthome/blob/master/bundles/core/org.eclipse.smarthome.core/src/main/java/org/eclipse/smarthome/core/library/types/DateTimeType.java#L48 seems fine.